edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/ConnectionMetadata.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core
+{
+    public class ConnectionMetadata
+    {
+        public string ModelId { get; set; }
+
+        public string ProductInfo { get; set; }
```

Comment:
don't these need to be JsonProperties?

Reply:
I guess not, because the tests pass, but it can't hurt to add them

Reply:
You can remove JsonProperty.. JsonProperty should only be needed if the serialized name is different.